nomad/rabbitmq.nomad

```diff
@@ -70,6 +77,16 @@ job "rabbitmq" {
         EOF
       }
 
+      template {
+        destination   = "local/rabbitmq-env.conf"
+        change_mode   = "signal"
+        change_signal = "SIGHUP"
+        data          = <<-EOF
+            USE_LONGNAME=true
+            NODENAME="rabbit@$(hostname).node.consul"
```

Comment:
Or use a fancy template value (`{{ ... }}`) here if one is available with the short host name.